lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -383,6 +389,17 @@ class ReportUIFeatures {
 
   closeToolsDropdown() {
     this.toolsButton.classList.remove('active');
+    this.toolsButton.setAttribute('aria-expanded', 'false');
+  }
+  
+  openToolsDropdown() {
+      this.toolsButton.classList.add('active');
+      this.toolsButton.setAttribute('aria-expanded', 'true');
+      this._document.addEventListener('keydown', this.onKeyDown);
+      // Wait for the first child to be eligible for focus
+      window.setTimeout(() => {
+        this._dom.find(':first-child', this.toolsDropDown).focus();
```

Comment:
This is also a messy way to get the first element. I am thinking it might be better to dynamically build the menu in JS keeping references around.

Thoughts?

Reply:
It was decided that keeping state in the DOM is okay 👍